app/javascript/packs/document-capture.tsx

```diff
@@ -126,6 +130,8 @@ const App = composeComponents(
         inPersonOutageExpectedUpdateDate,
         inPersonFullAddressEntryEnabled: inPersonFullAddressEntryEnabled === 'true',
         usStatesTerritories: parsedUsStatesTerritories,
+        skipDocAuth: skipDocAuth === 'true',
```

Comment:
TypeScript n00b question: We define it as a boolean in the previous file, but here it's a string. Is this in error? Or is it just two separate uses of the same name?

Reply:
skipDocAuth comes in as a string when we pass in value from Ruby to React.  I am using the eval to convert it back to a boolean - so we can pass around/evaluate a boolean in React. (All of the tsx files should now define it/expect it as a boolean)